src/ORM/Table.php

```diff
@@ -1989,6 +1989,13 @@ protected function _update($entity, $data)
             return $entity;
         }
 
+        if (empty($primaryColumns)) {
```

Comment:
Are there values that could be falsy that would accidentally trigger this to be true?

Reply:
I dont think so.

Reply:
No it should always be a list of columns.

Reply:
I'll use `count()` instead